keyboards/vortex/m0110a/info.json

```diff
@@ -0,0 +1,488 @@
+                    { "label": "=", "matrix": [6, 6], "x": 12, "y": 0 },
+                    { "label": "delete", "matrix": [6, 5], "x": 13, "y": 0 },
+                    { "label": "Backspace", "matrix": [10, 1], "x": 14, "y": 0},
+
+                    { "label": "Numlock", "matrix": [11, 4], "x": 15.5, "y": 0 },
+                    { "label": "Num/", "matrix": [12, 4], "x": 16.5, "y": 0 },
+                    { "label": "Num*", "matrix": [13, 4], "x": 17.5, "y": 0 },
+                    { "label": "Num-", "matrix": [13, 5], "x": 18.5, "y": 0 },
+                    { "label": "Tab", "matrix": [1, 1], "w": 1.5, "x": 0, "y": 1 },
+                    { "label": "Q", "matrix": [1, 0], "x": 1.5, "y": 1 },
+                    { "label": "W", "matrix": [2, 0], "x": 2.5, "y": 1 },
+                    { "label": "E", "matrix": [3, 0], "x": 3.5, "y": 1 },
+                    { "label": "R", "matrix": [4, 0], "x": 4.5, "y": 1 },
+                    { "label": "T", "matrix": [4, 1], "x": 5.5, "y": 1 },
+                    { "label": "Y", "matrix": [5, 1], "x": 6.5, "y": 1 },
+                    { "label": "U", "matrix": [5, 0], "x": 7.5, "y": 1 },
+                    { "label": "I", "matrix": [6, 0], "x": 8.5, "y": 1 },
+                    { "label": "O", "matrix": [7, 0], "x": 9.5, "y": 1 },
+                    { "label": "P", "matrix": [8, 0], "x": 10.5, "y": 1 },
+                    { "label": "[", "matrix": [8, 1], "x": 11.5, "y": 1 },
+                    { "label": "]", "matrix": [6, 1], "x": 12.5, "y": 1 },
+                    { "label": "\\", "matrix": [10, 5], "w": 1.5, "x": 13.5, "y": 1 },
+                    { "label": "Num7", "matrix": [11, 0], "x": 15.5, "y": 1 },
+                    { "label": "Num8", "matrix": [12, 0], "x": 16.5, "y": 1 },
+                    { "label": "Num9", "matrix": [13, 0], "x": 17.5, "y": 1 },
+                    { "label": "Num+", "matrix": [11, 3], "h":2, "x": 18.5, "y": 1 },
+
+                    { "label": "Caps Lock", "matrix": [2, 1], "w": 1.75, "x": 0, "y": 2 },
+                    { "label": "A", "matrix": [1, 2], "x": 1.75, "y": 2 },
+                    { "label": "S", "matrix": [2, 2], "x": 2.75, "y": 2 },
+                    { "label": "D", "matrix": [3, 2], "x": 3.75, "y": 2 },
+                    { "label": "F", "matrix": [4, 2], "x": 4.75, "y": 2 },
+                    { "label": "G", "matrix": [4, 3], "x": 5.75, "y": 2 },
+                    { "label": "H", "matrix": [5, 3], "x": 6.75, "y": 2 },
+                    { "label": "J", "matrix": [5, 2], "x": 7.75, "y": 2 },
+                    { "label": "K", "matrix": [6, 2], "x": 8.75, "y": 2 },
+                    { "label": "L", "matrix": [7, 2], "x": 9.75, "y": 2 },
+                    { "label": ";", "matrix": [8, 2], "x": 10.75, "y": 2 },
+                    { "label": "'", "matrix": [8, 3], "x": 11.75, "y": 2 },
+                    { "label": "Enter", "matrix": [10, 4], "w": 2.25, "x": 12.75, "y": 2 },
```

Comment:
Cleaning up the keymaps a bit.

Current implementation:
![vortex_m0110a](https://github.com/qmk/qmk_firmware/assets/18669334/1440f32b-8f50-419b-97a9-662ca72f15b7)

```suggestion
                    { "label": "#", "matrix": [0, 2], "x": 12.75, "y": 2 },
                    { "label": "Enter", "matrix": [10, 4], "w": 1.25, "x": 13.75, "y": 2 },
```